Makefile.standard_app

```diff
@@ -60,6 +60,14 @@ else
         DEFINES += PRINTF\(...\)=
 endif
 
+#####################################################################
+#                          LEDGER_ASSERT                            #
+#####################################################################
+ifneq ($(DISABLE_LEDGER_ASSERT), 1)
+	DEFINES += HAVE_LEDGER_ASSERT_DISPLAY
```

Comment:
Are we sure we want this by default? Exposing end user of weird info (rather than an app exit without info, maybe we need a product on this?)

Reply:
Well, I am not really fond of this also, but I thought that was what the majority wanted :woman_shrugging: 
I'll just keep it inside debug then :)